src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -225,12 +233,42 @@ private void initLayout() {
         head.setSpacing(0d);
         setTop(head);
 
-        splitPane.getItems().addAll(tabbedPane);
+        boolean hasPreviousFiles = !stateManager.getOpenDatabases().isEmpty();
+
+        if (!hasPreviousFiles) {
```

Comment:
Merge into one line. The variable is not used further down, is it?

(The variable name is pretty bad - it should be `openedDatabasesExisting`.